redisinsight/desktop/src/lib/server/server.ts

```diff
@@ -23,24 +24,71 @@ let gracefulShutdown: Function
 let beApp: any
 export const launchApiServer = async () => {
   try {
+    console.log('Launching API server', process.env.NODE_ENV, process.env.RI_APP_TYPE, process.env.NODE_ENV === 'development' && process.env.RI_APP_TYPE === 'electron')
+    if (process.env.NODE_ENV === 'development' && process.env.RI_APP_TYPE === 'electron') {
```

Comment:
there is only server 'electron', no need to check `process.env.RI_APP_TYPE === 'electron'`